obal/data/tasks/download_and_extract_source.yml

```diff
@@ -114,26 +42,27 @@
       replace:
         path: "{{ spec_file_path }}"
         regexp: '^(?P<prefix>{{ item }}\s+)[0-9.]+(?P<suffix>.*)$'
-        replace: '\g<prefix>{{ downstream_release }}\g<suffix>'
+        replace: '\g<prefix>{{ release }}\g<suffix>'
       with_items:
         - 'Release:'
         - '%global _release'
         - '%global release'
 
+    - name: 'Extract packager info'
+      command: "rpmdev-packager"
+      register: packager_info
+
     - name: 'add changelog entry'
       lineinfile:
         path: "{{ spec_file_path }}"
         line: |
-          * {{ lookup('pipe', 'date +"%a %b %d %Y"') }} {{ downstream_changelog_name | default('Satellite6 Jenkins') }} {{ downstream_version }}-{{ downstream_release }}
-          {{ downstream_changelog }}
+          * {{ lookup('pipe', 'date +"%a %b %d %Y"') }} {{ packager_info.stdout }} {{ version }}-{{ release }}
```

Comment:
For those of us with different locales it might be good to explicitly set ```$LANG```

Reply:
What will that affect?

Reply:
```
$ LANG=nl_NL.UTF-8 date +"%a %b %d %Y"
do mrt 22 2018
```